lib/facilities/drivetime_bands/configuration.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require 'common/client/configuration/rest'
+require 'common/client/middleware/response/raise_error'
+
+module Facilities
+  module DrivetimeBands
+    class Configuration < Common::Client::Configuration::REST
+      def base_path
+        Settings.locators.drive_time_band_base_path
+      end
+
+      def service_name
+        'FL'
```

Comment:
Just curious -- where did that name/abbreviation come from?

Reply:
I think it might stand for Facility Locator?

Reply:
There's a job that does some Geo coding on drivetime bands for facilities

Reply:
When we were creating this we decided to stick with "FL" to match the existing facilities services, but would be happy to see it replaced with a more descriptive name at some point.